### ISSUE TITLE
2.10.0 broken

### ISSUE THREAD

**Issue by ehelms** [patternfly/patternfly-sass#100]
The latest 2.10.0 release changed expectations for some SCSS files and breaks projects using it because sprockets cannot find, for example 'bootstrap-combobox'. Looking through the recent commit log the change (https://github.com/patternfly/patternfly-sass/commit/7f9fd91d6da86492488583a64b93fb4cbfad6a4e#diff-8dc22444ca25a5fcafa9be7c62b0d8f3L82) was made that moved away from the provided SCSS files for what appears to be existing gems. However, the patternfly-sass gem does not require those gems. I cannot tell if this was intentional or a bug.

**Comment by iNecas** [comment]
I can just confirm that it's broken for me as well.

**Comment by domcleal** [comment]
It appears to use rails-assets to load these components from Bower and stores them into the gem, under assets/stylesheets/patternfly/, replacing the version vendored into Patternfly itself.

The Patternfly converter though is putting the wrong path into the resulting _patternfly.scss, e.g.

    -@import "bootstrap-combobox";
    -@import "bootstrap-datepicker/bootstrap-datepicker3";
    -@import "bootstrap-select";
    +@import "patternfly/bootstrap-combobox";
    +@import "patternfly/bootstrap-datepicker";
    +@import "patternfly/bootstrap-select";

I'm just prepping a patch.

**Comment by domcleal** [comment]
Sorry, I don't think I can prepare a complete patch for this, I'll have to leave it to the maintainers.  I got as far as: https://github.com/domcleal/patternfly-sass/commit/f6f6acd7b9b44e9a1490c1649ae97941112e8de9, but the repo is missing a c3 file (only ext_c3, which is copied from patternfly itself).

As far as I can tell, these components (such as datepicker, combobox) are still from the external rails-assets/bower gems and meant to be distributed in patternfly-sass.  The code to copy them (https://github.com/patternfly/patternfly-sass/blob/3c826848e59f1f86cc1572bcae726284e98b619c/tasks/converter.rb#L135) was removed though as @ehelms points out.  Perhaps it was intentional, but that implies there would have to be a hard runtime dependency on rails-assets for users.

It seems that c3 needs to be copied in too from rails-assets-c3, but since I can't find any rails-assets copying code, I'm not sure where to add it (except to do it manually).
